daemon/daemon.go

```diff
@@ -144,40 +151,50 @@ func (d *Daemon) DriverPool(ctx context.Context, language string) (*DriverPool,
 
 	d.mu.Lock()
 	defer d.mu.Unlock()
+	if l, ok := d.aliases[language]; ok {
+		language = l
+	}
 	dp, ok = d.pool[language]
 	if ok {
 		return dp, nil
 	}
 
-	image, err := d.getDriverImage(ctx, language)
+	image, aliases, err := d.getDriverImage(ctx, language)
 	if err != nil {
 		return nil, ErrRuntime.Wrap(err)
 	}
 
-	return d.newDriverPool(ctx, language, image)
+	return d.newDriverPool(ctx, language, aliases, image)
 }
 
-func (d *Daemon) getDriverImage(rctx context.Context, language string) (runtime.DriverImage, error) {
+func (d *Daemon) getDriverImage(rctx context.Context, language string) (runtime.DriverImage, []string, error) {
 	sp, _ := opentracing.StartSpanFromContext(rctx, "bblfshd.runtime.ListDrivers")
 	defer sp.Finish()
 
 	list, err := d.runtime.ListDrivers()
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
 	for _, d := range list {
```

Comment:
As this code looks very structural similar to [`containsLang()`](https://github.com/bblfsh/sdk/pull/400/files#diff-c7921d573f39a395c057e7eeb48f78daR61) - would it be worth to extract a function here as well?